index.js

```diff
@@ -28,10 +34,14 @@ function registerListener(win, opts = {}, cb = () => {}) {
 			}
 
 			if (state === 'interrupted') {
-				const message = `The download of ${item.getFilename()} was interrupted`;
-				electron.dialog.showErrorBox('Download error', message);
+				const message = errorMessage.replace('FILENAME', item.getFilename());
```

Comment:
Use `{filename}` as the placeholder and use https://github.com/sindresorhus/pupa to expand it.